### ISSUE TITLE
[HELP]MQTT connector attributeUpdates problem

### ISSUE THREAD

**Issue by coder7675** [thingsboard/thingsboard-gateway#1119]
**Describe the issue**

I am learning and using the gateway, and I tested the connectRequests, disconnectRequests, and attributeRequests sections, and it worked properly. However, the attributeUpdates section does not work. It seems that the gateway did not receive any messages from tb. Did I miss something? How should I troubleshoot the problem?

Here are some screenshots of the details:
![image](https://github.com/thingsboard/thingsboard-gateway/assets/67080835/0f13cea7-ce41-4a27-aa97-82f9e4d81a41)
![image](https://github.com/thingsboard/thingsboard-gateway/assets/67080835/74087ba4-8bf9-4dd7-af2a-ee8049df3d60)
![image](https://github.com/thingsboard/thingsboard-gateway/assets/67080835/c2087a2e-d51f-42b1-85ee-422513553057)
![image](https://github.com/thingsboard/thingsboard-gateway/assets/67080835/4a65554f-2789-4e85-9958-6fc9197bc061)

When the value is updated, gateway does not receive any messages

**Configuration**
```   
// mqtt.json

  "attributeUpdates": [
    {
      "retain": false,
      "deviceNameFilter": "SN-001",
      "attributeFilter": "FirmwareVersion",
      "topicExpression": "/sensor/${deviceName}/${attributeKey}",
      "valueExpression": "{\"${attributeKey}\":\"${attributeValue}\"}"
    }
  ]
````

**Connector name:**
MQTT Connector

**Versions (please complete the following information):**
 - OS: [Win11]
 - Thingsboard IoT Gateway version [3.2]
 - Python version[3.9]
 - Thingsboard CE version [3.4.0]

Thank you,Best regards.

**Comment by coder7675** [comment]
I found that if the device is connected and then the gateway restarts, when the device try to request connected again, TB is not connected to the device unless the device request disconnected first. I am not sure if this behavior is normal. I solved my problem by modifying the code for method add_device.

**Comment by imbeacon** [comment]
Hi @coder7675 ,
Thank you for your message and investigation, we will check this point.

**Comment by imbeacon** [comment]
The issue relates to the device activity and it was caused by removing the device session on the ThingsBoard side, it was fixed in ThingsBoard 3.7, but if your device doesn't publish data more often than once per 5 minutes - it is recommended to enable "Overwrite device activity for connected devices" parameter on UI for the gateway device.
